<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TrustedTest</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>App.png</normaloff>App.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1001</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
	background-color: rgb(170, 207, 255);
}</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>601</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap>Back_2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>531</width>
      <height>380</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_2{
	background-color: rgb(195, 250, 177);
	border-radius:25px;
}</string>
    </property>
    <widget class="QLineEdit" name="le_number">
     <property name="geometry">
      <rect>
       <x>288</x>
       <y>165</y>
       <width>216</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter your Phone Number</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border-radius:10px;
    background-color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border-radius:10px;
	background-color: rgb(210, 237, 238);
}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_aadhar">
     <property name="geometry">
      <rect>
       <x>288</x>
       <y>97</y>
       <width>216</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter your Aadhar Card Number</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border-radius:10px;
    background-color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border-radius:10px;
	background-color: rgb(210, 237, 238);
}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_pass">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>165</y>
       <width>216</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Should contain atleast 1 Upppercase, 1 number and 1 special character, maxlength is 20</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border-radius:10px;
    background-color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border-radius:10px;
	background-color: rgb(210, 237, 238);
}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>73</y>
       <width>68</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>216</x>
       <y>5</y>
       <width>136</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Registration</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>140</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>282</x>
       <y>209</y>
       <width>44</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>E-Mail</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_signup">
     <property name="geometry">
      <rect>
       <x>216</x>
       <y>305</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 14pt &quot;Calibri&quot;;
	border-radius:15px;
	color: rgb(0, 0, 0);
	background-color: rgb(150, 238, 238);
}
QPushButton:hover{
	font: 14pt &quot;Calibri&quot;;
	border-radius:15px;
	color: rgb(0, 0, 127);
	background-color: rgb(189, 238, 238);
}
</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_mail">
     <property name="geometry">
      <rect>
       <x>288</x>
       <y>232</y>
       <width>216</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter your Email-Id</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border-radius:10px;
    background-color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border-radius:10px;
	background-color: rgb(210, 237, 238);
}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_name">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>97</y>
       <width>216</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter a Username</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border-radius:10px;
    background-color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border-radius:10px;
	background-color: rgb(210, 237, 238);
}</string>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
    <widget class="QPushButton" name="but_login">
     <property name="geometry">
      <rect>
       <x>471</x>
       <y>344</y>
       <width>38</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
font: bold 12pt &quot;Calibri&quot;;
border-radius:10px;
color:rgb(0, 0, 127)
}
QPushButton:hover{
font: bold 12pt &quot;Calibri&quot;;
border-radius:10px;
color:rgb(0, 0, 255)
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>139</y>
       <width>119</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Phone Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>281</x>
       <y>73</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Aadhar Card No.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>384</x>
       <y>345</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Already a User?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>210</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Year of Birth</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_dob">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>230</y>
       <width>216</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>ENter your Year of BIrth</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border-radius:10px;
    background-color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border-radius:10px;
	background-color: rgb(210, 237, 238);
}</string>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="lab_error1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>272</y>
       <width>340</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Calibri&quot;;
text-decoration: underline;
color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
